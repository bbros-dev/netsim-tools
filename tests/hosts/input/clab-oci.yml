provider: clab
name: hosts
defaults:
  device: frr

nodes:
- name: h1
- name: h2

links:
- h1-h2
