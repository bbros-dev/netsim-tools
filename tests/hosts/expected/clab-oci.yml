# Ansible inventory created from ['tests/hosts/input/clab-oci.yml', 'package:topology-defaults.yml']
#
---
frr:
  hosts:
    h1:
      ansible_host: clab-hosts-h1
      connection_type: docker
      id: 1
    h2:
      ansible_host: clab-hosts-h2
      connection_type: docker
      id: 2
